justfile

```diff
@@ -0,0 +1,67 @@
+set quiet # Recipes are silent by default
+set export # Just variables are exported to environment variables
+
+default_component := "temporal-server"
+
+[private]
+default:
+	just --list
+
+[private]
+push-to-registry component:
+	#!/usr/bin/bash
+	set -e
+	echo "Pushing ${component} to local"
+	rock_file=$(ls "${component}" | grep "\.rock")
+	rock_version=$(yq ".version" "${component}/rockcraft.yaml")
+	rockcraft.skopeo --insecure-policy copy --dest-tls-verify=false \
+		"oci-archive:${component}/${rock_file}" \
+		"docker://localhost:32000/${component}-dev:${rock_version}"
+
+# Clean the rockcraft project of the component and remove any existing rock files
+clean component=default_component:
+	cd "${component}" && rockcraft clean
+	cd "${component}" && rm -f "$(ls | grep *.rock)"
+	rm -rf "${component}/temporal"
+
+# Pack a rock for a specific component
+pack component=default_component:
```

Comment:
Is it strictly necessary to have a default component? Should the default behaviour be to prompt a message to pass a component instead?